src/handlers.js

```diff
@@ -339,21 +354,23 @@ async function jobHandler(message) {
           '```\n',
           '</details>',
         ].join('\n');
-        await instGithub.repos.createCommitComment({
+        //console.log(instGithub.pullRequests);
+        //console.log('..pr number' organization);
+        await instGithub.pullRequests.createComment({
           owner: organization,
           repo: repository,
-          sha,
+          number,
           body,
         });
         return;
       }
     } catch (e) {
       if (e.name === 'YAMLException') {
         let docsLink = 'https://docs.taskcluster.net/reference/integrations/github/docs/usage#who-can-trigger-jobs';
-        await instGithub.repos.createCommitComment({
+        await instGithub.pullRequests.createComment({
```

Comment:
We only want to use `createComment` instead of `createCommitComment` if this is a pull request.